modules/ROOT/pages/config-add-proxy-whitelist.adoc

```diff
@@ -27,3 +27,53 @@ The following example defines regular expressions that allow requests to be made
 ----
 kubectl delete pod -n api-manager -l component=api-platform-web
 ----
+
+NOTE: This method is currently working from version 3.2.0 to version 3.2.1. From version 3.2.2 and any future versions, refer to Trusted Domains.
+
+== Trusted Domains
+
+Every HTTP request you make to an external resource goes through the API Console Proxy. For security reasons, you need to register the domain as an allowed domain in Anypoint Platform. 
+
+By default, proxy requests are blocked for external domains on authenticated requests.
+
+To add a new domain:
+
+1. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+
+2. In the navigation bar or the main Anypoint Platform page, click *Access Management*.
+
+3. Click *Trusted domains*.
+
+4. Click *Add domain*. +
+The domain must not match with any existing domain,  must not be any domain or subdomain of MuleSoft or Anypoint Platform, and should be written in the format `exampledomain.com`.
+NOTE: Regular expressions are not supported at the moment.
+
+5. In the dialog box that appears, type the domain name and click Add domain. +
+
+Result: The domain has been added to your list of trusted domains. You are redirected to the trusted domains page, where you can edit or delete your domains. If you register a domain on a parent organization, it is also allowlisted on all your business groups.
+
+== Authentication 
+
+When using the proxy, there are two authentication modes available: 
+
+*Strict Authentication*: All requests reaching the proxy must be authenticated. This means that user token credentials must be included in the ms-authorization header of the request. This is the default option.
+
+WARNING: As requests to external domains from the Exchange Public Portals go through the API Console Proxy, they will fail as no Anypoint user is involved in the call.
```

Comment:
```suggestion
WARNING: If requests to external domains from the Exchange Public Portals go through the API Console Proxy, they fail because no Anypoint user is involved in the call.
```